cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -8,7 +8,7 @@
 {% block hero -%}
     {% for block in page.header -%}
         {% import 'templates/render_block.html' as render_block with context %}
-        {{ render_block.render(block, loop.index) }}
+        {{ render_block.render(block) }}
```

Comment:
This is wrong -- will change back, as well as all other updates to form_block and render_block that I incorrectly made.